source/platforms/github/_tests/_github_git.test.ts

```diff
@@ -20,15 +20,19 @@ export const requestWithFixturedContent = async (path: string): Promise<() => Pr
   Promise.resolve(readFileSync(`${fixtures}/${path}`, {}).toString())
 )
 
+const pullRequestInfoFilename = "github_pr.json"
+const masterSHA = JSON.parse(readFileSync(`${fixtures}/${pullRequestInfoFilename}`, {}).toString()).base.sha
```

Comment:
try using the path API for this string - that might be what's breaking on windows, can't remember its name yet